.github/workflows/default.yml

```diff
@@ -0,0 +1,67 @@
+name: run-tests
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - master
+
+jobs:
+  main:
+    name: Run all tests
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - compiler: dmd-latest
+            dmd: dmd
+
+          - compiler: ldc-latest
+            dmd: ldmd2
+
+          - compiler: gdc-latest
+            dmd: gdmd
```

Comment:
I don't think we support GDC right now because of using newer features, unless gdc-latest is already the newer one with all the cool new features. Did this work on your repo?